src/ast-types/smdast/element.ts

```diff
@@ -0,0 +1,10 @@
+import * as Unist from '@core/unist';
```

Comment:
Legacy `@core` import. However, not sure if we should move these ast-types over until we have a better idea how we'll use them in this package.